cmd/sync/sync.go

```diff
@@ -23,24 +23,8 @@ var (
 	logLevel = flag.String("loglevel", "Debug", "valid values are Debug, Info, Warning, Error")
 )
 
-// checks and sanitizes logLevel input
-func sanitizeLogLevel(lvl *string) log.Level {
-	switch strings.ToLower(*lvl) {
-	case "debug":
-		return log.DebugLevel
-	case "info":
-		return log.InfoLevel
-	case "warning":
-		return log.WarnLevel
-	case "error":
-		return log.ErrorLevel
-	default:
-		// silently default to info
-		return log.InfoLevel
-	}
-}
 func sync() error {
-	log.Print("Sync process started!")
+	logrus.Print("Sync process started!")
```

Comment:
Any particular reason you use logrus directly and not via wrapper `log.Stuff`? Same as other cmd packages?

Reply:
We used logrus directly even before my changes.

Reply:
in sync I think because it was under development. All other code should be using log from wapper now. Wrapper main idea was not to pass `hc.log.stuff`, `p.log.stuff` around in the structs and just use it like this https://github.com/openshift/openshift-azure/blob/master/pkg/plugin/plugin.go#L42

Reply:
Sure, although I dislike globals (and every logging library has already its own global), I am not planning on doing such a change here.

Reply:
Whats the alternative? We pass `entry` file down the pipe to each our subpackage in the structs?